JULIERAJ/KIDS-FIRST-v2
# Fix/127/warnings workflow

## Fix Warnings, Adjust Workflow, Refactor Redirection, Buttons

## Requirements
- Fix warnings in the calendar console
- Delete unneeded steps from workflow
- Check buttons' hover in password reset and page after verification
- Refactor: useNavigate instead of window.location.href

## Test Steps
- Click on create button on the calendar page - there should be no warnings in the console
- After sign in/up you should be redirected to the dashboard
- Buttons in password reset and after verification should have correct hover (like in sign in/up)

## Notes
New devs names added

## Checklist
- Pull Request title includes the issue number and a brief description of the change.
- All changes should be tested and verified to work correctly.
- Code follows frontend best practices.
- Branch names follow the conventions in the contributing file.
- Commit messages follow the naming conventions in the contributing file.
